### PR TITLE
custom-conf-parameters&pv-claim&balance-data

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

File: docs-2.0/nebula-operator/8.custom-cluster-configurations/8.2.pv-reclaim.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+The following example uses a cluster named `nebula` to show how to set `enablePVReclaim`:
+
+1.  Run the following command to access the edit page of the `nebula` cluster.
+   
+  ```bash
+  kubectl edit nebulaclusters.apps.nebula-graph.io nebula
+  ```
+
+2. In the `spec` configuration item of the `nebula` cluster YAML file, add `enablePVReclaim` and set its value to `true`.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/8.custom-cluster-configurations/8.2.pv-reclaim.md:23]: Same with above.

File: docs-2.0/nebula-operator/8.custom-cluster-configurations/8.3.balance-data-when-scaling-storage.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# Balance storage data after scaling out
+
+After the Storage service is scaled out, you can decide whether to balance the data in the Storage service. 
+
+The scaling out of the Nebula Graph's Storage service is divided into two stages. In the first stage, the status of all pods is changed to `Ready`. In the second stage, the commands of `BALANCE DATA`和`BALANCE LEADER` are executed to balance data. These two stages decouple the scaling out process of the controller replica from the balancing data process, so that you can choose to perform the data balancing operation during low traffic. The decoupling of the scaling out process from the balancing process can effectively reduce the impact on online services during data migration.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/8.custom-cluster-configurations/8.3.balance-data-when-scaling-storage.md:5]: during low traffic => During low traffic period or In off-peak hours.

File: docs-2.0/nebula-operator/8.custom-cluster-configurations/8.3.balance-data-when-scaling-storage.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+You can define whether to balance data automatically with the parameter `enableAutoBalance` in the configuration file of the CR instance of the cluster you created.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/8.custom-cluster-configurations/8.3.balance-data-when-scaling-storage.md:7]: Check the usage of whether to again.

File: docs-2.0/nebula-operator/8.custom-cluster-configurations/8.3.balance-data-when-scaling-storage.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+## Prerequisites
+
+You have created a cluster. For how to create a cluster with Kubectl, see [Create a cluster with Kubectl](../3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md).
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/8.custom-cluster-configurations/8.3.balance-data-when-scaling-storage.md:11]: You have created a Nebula Graph cluster.

File: docs-2.0/nebula-operator/8.custom-cluster-configurations/8.3.balance-data-when-scaling-storage.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+The scaling out of the Nebula Graph's Storage service is divided into two stages. In the first stage, the status of all pods is changed to `Ready`. In the second stage, the commands of `BALANCE DATA`和`BALANCE LEADER` are executed to balance data. These two stages decouple the scaling out process of the controller replica from the balancing data process, so that you can choose to perform the data balancing operation during low traffic. The decoupling of the scaling out process from the balancing process can effectively reduce the impact on online services during data migration.
+
+You can define whether to balance data automatically with the parameter `enableAutoBalance` in the configuration file of the CR instance of the cluster you created.
+
+## Prerequisites
+
+You have created a cluster. For how to create a cluster with Kubectl, see [Create a cluster with Kubectl](../3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md). 
+
+## Step
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/8.custom-cluster-configurations/8.3.balance-data-when-scaling-storage.md:13]: Steps

File: docs-2.0/nebula-operator/8.custom-cluster-configurations/8.3.balance-data-when-scaling-storage.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+The following example uses a cluster named `nebula` to show how to set `enableAutoBalance`.
+
+1. Run the following command to access the edit page of the `nebula` cluster.
+   
+  ```bash
+  kubectl edit nebulaclusters.apps.nebula-graph.io nebula
+  ```
+
+2. In the `spec.storaged` configuration item of the `nebula` cluster YAML file, add `enableAutoBalance` and set its value to `true`.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/8.custom-cluster-configurations/8.3.balance-data-when-scaling-storage.md:23]: Same as above.

File: docs-2.0/nebula-operator/8.custom-cluster-configurations/8.1.custom-conf-parameter.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
+# Custom configuration parameters for a Nebula Graph cluster
```
[the remark this answers is not in the document]
Comment (abby-cyber) [reply docs-2.0/nebula-operator/8.custom-cluster-configurations/8.1.custom-conf-parameter.md:1]: hmmm, custom can be used as a verb
both custom and customize can be used. 
To unify our style, I will use customize instead.

File: docs-2.0/nebula-operator/8.custom-cluster-configurations/8.2.pv-reclaim.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
+# Reclaim PV
+
+Nebula Operator uses PVs (Persistent Volume) and PVCs (Persistent Volume Claim) to store persistent data. If you accidentally deletes a Nebula Graph cluster, PV and PVC objects and the relevant data will be retained to ensure data security.
```
[the remark this answers is not in the document]
Comment (abby-cyber) [reply docs-2.0/nebula-operator/8.custom-cluster-configurations/8.2.pv-reclaim.md:3]: yeah, thank you for this note.

**Review by randomJoe211 — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**